webui/react/src/components/Link.tsx

```diff
@@ -40,7 +41,9 @@ const Link: React.FC<Props> = ({
     handlePath(event, { external, onClick, path: props.path, popout });
   }, [ onClick, popout, props.path, external ]);
 
-  return props.disabled ? (
+  return props.disabled ? props.isButton ? (
```

Comment:
nit: maybe add parens or something to make the nested ternary easier to read?

Reply:
👍🏼  I removed one of the ternaries